Add more information on working on cluster attached to ESGF node (#1821)
ESMValGroup/ESMValTool: doc/sphinx/source/input.rst

Obtaining input data
********************

ESMValTool supports input data from climate models participating in
`CMIP6 <https://www.wcrp-climate.org/wgcm-cmip/wgcm-cmip6>`__,
`CMIP5 <https://www.wcrp-climate.org/wgcm-cmip/wgcm-cmip5>`__,

> `CMIP3 <https://www.wcrp-climate.org/wgcm-cmip/wgcm-cmip6>`__, and

Reviewer: ```suggestion
`CMIP3 <https://www.wcrp-climate.org/wgcm-cmip/wgcm-cmip3>`__, and
```

but many more exist around the world.

If you do not have access to such a facility through your institute or the
project you are working on, you can request access by applying for the
`IS-ENES3 Trans-national Access call <https://portal.enes.org/data/data-metadata-service/analysis-platforms>`__.

If the options above are not available to you, ESMValTool also offers features
to make it easier to download the data.

Models
======

ESMValTool will look for existing data in the directories specified in the
user configuration file. Alternatively, it can use an external
tool called `Synda <http://prodiguer.github.io/synda/index.html>`__. If you
do not have access to a compute cluster with the data already mounted, this is
the recommended approach for first-time users to obtain some data for
running ESMValTool.

Installing Synda for use from ESMValTool
----------------------------------------
Here, we describe the basic steps to configure EMSValTool so it can use Synda
to download CMIP6 or CMIP5 model data.

To install Synda, follow the steps listed in the
`Synda installation documentation <http://prodiguer.github.io/synda/sdt/conda_install.html>`__.

> This description assumes that you use the conda installation.

Reviewer: ```suggestion
(This description assumes that Synda is installed using Conda.)
```
